test/components/telemetry_test.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react"
+import { render, waitFor } from "@testing-library/react-native"
+import { act } from "react-test-renderer"
+
+import {
+  ConnectWidget as BaseConnectWidget,
+  ConnectWidgetProps,
+} from "../../src/components/ConnectWidgets"
+import { SdkTelemetryProps } from "../../src/components/telemetry"
+import { sdkVersion } from "../../src/version"
+
+/* The exported widgets doesn't specify that they adhere to the
+ * SdkTelemetryProps type (because we don't want to clutter their prop types)
+ * but we _know_ that they do, so we cast ConnectWidget before using it. Since
+ * the casting syntax conflicts with JSX, we first put it in a variable and
+ * then use that variable in the code.
+ */
+const TestConnectWidget = BaseConnectWidget as FC<ConnectWidgetProps & SdkTelemetryProps>
```

Comment:
I'm gonna do the same with the `OAuthProps` in https://github.com/mxenabled/react-native-widget-sdk/blob/master/test/components/widgets_test.tsx.